src/api/Visualization/index.ts

```diff
@@ -400,4 +402,16 @@ export class VisualizationAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Delete visualization comment
+   */
+  deleteComment(params: DeleteVisualizationCommentParams) {
```

Comment:
Since the APIs for creating and getting comments are named 
`createVisualizationComment`
`getVisualizationComments`

lets follow the same pattern and rename this function to
`deleteVisualizationComment`